routers/api/v1/repo/repo.go

```diff
@@ -983,6 +984,89 @@ func Delete(ctx *context.APIContext) {
 	ctx.Status(http.StatusNoContent)
 }
 
+// Transfer transfers the ownership of a repository
+func Transfer(ctx *context.APIContext, opts api.TransferRepoOption) {
```

Comment:
Move this function to a separate file repo_transfer.go we should avoid files going over 1K lines...